parsers/mcmaster.js

```diff
@@ -1,283 +1,146 @@
-		}
-		fs.writeFile(directory + filename, icsFileContent)	
-		callback(null, filename);
-	});		
+var ics = require('./ics-builder');
+var fs = require('fs')
+    , filename = process.argv[2];
+/**
+ * Takes in any time and converts it from a 12HR format to the 24HR format for 7:00PM is 19:00 etc
+ * @param {String} time
+ * @return {String} 24HR formatted time
+ */
+function convertTo24Hour(time) {
+    var hours = parseInt(time.substr(0, 2), 10);
+    if(time.indexOf('AM') > -1 && hours == 12) {
+        time = time.replace('12', '0');
+    }else if(time.indexOf('PM')  > -1 && hours < 12) {
+        time = time.replace(hours, (hours + 12));
+    }
+
+    return time.replace(/(AM|PM)/, '');
 }
 
-/*
-dayString: Put in Monday - Sunday.
-Uses to get the time in a format that an ical file can read. It take in the
-class time and adds 12 hours is the time has a PM
-Example: YYYY,MM,DD,HH,MM   -   [2015,08,07,09,30] is monday sept 7th at 9 30am
-*/
-var start_end_time = function(classtime,dayString,start_date) {
-		//get the time
-		//store the start and end time
-		require("datejs");
-		classtime = classtime.toString().split("-")
-		//start_date = [2015,8,7] //when school starts
-		//end_date = [2015,11,08] //when school ends
-		times = []
-		hour = [] //start and stop}
-
-		if (classtime[0].indexOf("PM") > 0){
-		 hour[0] = parseInt(classtime[0].split(":")[0]) + parseInt("12");
-			if (hour[0] == 24){
-						hour[0] = 12;
-					}
-				}
-				else {
-					hour[0] = parseInt(classtime[0].split(":")[0]);
-				}
-				if (classtime[1].indexOf("PM") > 0){
-				hour[1] = parseInt(classtime[1].split(":")[0]) + parseInt("12");
-					if (hour[1] == 24){
-						hour[1] = 12;
-					}
-				}
-				else {
-					hour[1] = parseInt(classtime[1].split(":")[0]);
-				}
-		
-		var AM_PM = /(\s*[A-Z]{2}\s*)/;
-  		classtime[0] = classtime[0].replace(AM_PM,""); //Match the details with the regualar exp
-		classtime[1] = classtime[1].replace(AM_PM,"");
-		stminute = classtime[0].split(":")[1];
-		etminute = classtime[1].split(":")[1];
-
-		date = Date.today().set({year:start_date[0],month:start_date[1],day:start_date[2]}) //.parse(dayString.toLowerCase())
-		if (dayString === "Monday"){
-			date = date.monday()
-		}
-		if (dayString === "Tuesday"){
-			date = date.tuesday()
-		}
-		if (dayString === "Wednesday"){
-			date = date.wed()
-		}
-		if (dayString === "Thursday"){
-			date = date.thursday()
-		}
-		if (dayString === "Friday"){
-			date = date.fri()
-		}
-		if (dayString === "Saturday"){
-			date = date.sat()
-		}
-		if (dayString === "Sunday") {
-			date = date.sunday()
-		}
-		times[0]=  [date.getFullYear(),date.getMonth(),date.getDate(), hour[0] , parseInt(stminute)];
-		times[1] = [date.getFullYear(),date.getMonth(),date.getDate(), hour[1] , parseInt(etminute)];
-		return times;
+/**
+ * Takes any date and turns it into the correct format, so javascript gets the right date.
+ * MM/DD/YYYY is the format the javascript object reads or YYYY/MM/DD
```

Comment:
put these two comments on the same line, no need to start a new line for a new sentence